test/lit.cfg

```diff
@@ -100,6 +100,8 @@ def get_simulator_command(run_os, run_cpu, sdk_path):
 
 def get_lldb_python_path(lldb_build_root):
     lldb_path = os.path.join(lldb_build_root, 'bin', 'lldb')
+    if not os.access(lldb_path, os.F_OK):
```

Comment:
Is there a reason this is preferable compared to something like ` os.path.exists`?

Reply:
No. I just did it this way out of habit.